src/Webapi/Dom/Webapi__Dom__History.res

```diff
@@ -1,13 +1,17 @@
 type t = Dom.history
-type state /* TODO: should be "anything that can be serializable" apparently */
+
+@ocaml.doc("State could be anything that is serializable, to use this convert manually using %identity externals")
```

Comment:
We have a `unsafeAsRadioNodeList` maybe we could have a `unsafeAsState`?

Reply:
I think for a type as critical as this custom code needs to be involved, if an identity shortcut is to be used it should be deliberate.

I was tempted to make it `Js.Json.t` but I worry that would be even harder to use than an abstract type.